python-nlp-service/bootstrap.py

```
@@ -1,15 +1,38 @@
 import os
 import subprocess
+import sys
+
+
+
+BASE_DIR = os.path.dirname(os.path.abspath(__file__))
+
+
+print("--- DEBUG INFO START ---")
+print(f"Current Working Directory: {os.getcwd()}")
+print(f"sys.path:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)